web/src/js/ducks/flows.ts

```diff
@@ -198,6 +204,15 @@ export function remove(flow: Flow) {
     return () => fetchApi(`/flows/${flow.id}`, { method: "DELETE" });
 }
 
+export function removeMultiple(flowIds: string[]) {
```

Comment:
I think the general pattern of adding separate "multi" actions isn't great. Why don't we change the existing `remove` action to always accept a list of flow ids? It's not used in a million places, we can easily adapt existing callsites. :)